### ISSUE TITLE
“Done” section: have world-viewable view?

### ISSUE THREAD

**Issue by kmcculloch** [edgi-govdata-archiving/archivers.space#19]
_From @khdelphine on February 9, 2017 20:6_

* Would show the final location of the bag in the case of a harvest
* And would show metadata fields (if available from Describe step)

_Copied from original issue: b5/pipeline#50_

**Comment by kmcculloch** [comment]
_From @b5 on February 9, 2017 21:52_

Oooooh I love this. I think we should make this public. Soonest we'll get to it is next week, but great idea!

**Comment by kmcculloch** [comment]
_From @khdelphine on February 9, 2017 23:36_

I am glad you like it! :-)

**Comment by kmcculloch** [comment]
@b5 @khdelphine Is this the same idea as https://github.com/edgi-govdata-archiving/archivers.space/issues/45 ?

**Comment by khdelphine** [comment]
Not necessarily. This is really about creating an end-user view. But I don't think this is high priority at this point since that's currently achieved through ckan: https://www.datarefuge.org/
